images/golang/Containerfile

```diff
@@ -1,7 +1,7 @@
 ARG ARCH
 FROM bootstrap-$ARCH
 
-ENV GIMME_GO_VERSION=1.22.6
+ENV GIMME_GO_VERSION=1.23.4
```

Comment:
Do we normally keep this inline with that is on the main branch of kubevirt?

Reply:
The comment suggests otherwise, however I'm not against it. OTOH I faintly remember the builder image pulling in it's own version?

Reply:
https://github.com/kubevirt/kubevirt/blob/main/hack/builder/Dockerfile#L8

Reply:
That differs already being 1.22.10 currently